pilot/pkg/proxy/envoy/v1/config.go

```diff
@@ -35,6 +35,12 @@ import (
 	"istio.io/istio/pkg/log"
 )
 
+var (
+	// validateClustersDefault is an environment variable that can be set to false to disable
+	// cluster validation in RDS, in case problems are discovered.
+	ValidateClustersDefault = true
```

Comment:
why is it called “default” when it’s going to be the value used, is there a way to set a different value that what get sets into this var (by the env)